doc/sphinxman/source/scf.rst

```diff
@@ -528,6 +528,40 @@ in a 3-21G basis and then casting up to cc-pVTZ is shown below::
 
     energy('scf')
 
+.. .. _`scf:restart`:
+
+Restarting the SCF
+~~~~~~~~~~~~~~~~~~
+
+Reading orbital data from a previous calculations is done via the ``restart_file`` option,
+where the actual file is a serialized ``wfn`` object (see :ref:`saving the wfn <sec:save_wfn>`)
+By default the orbital data file of the converged SCF(``psi.PID.name.180.npy``) is deleted
+after |PSIfour| exits or the ``clean()`` function is called.
+To keep the orbital data and also write the orbitals after every iteration the ``write_orbitals``
+options is available: ::
+
+  energy('scf', write_orbitals='my_mos'),
+
+which write a ``Wavefunction`` object converted (serialized) to a numpy file called ``my_mos.npy``.
+The restart can then be done as follows: ::
+
+  set guess read
+  energy('scf', restart_file='my_mos')
+
+Specifying the ``.npy`` suffix when writing and reading restart files is optional.
+
+Alternatively, the restart can also be done from any previously saved ``wfn`` object. ::
+
+  energy, scf_wfn = energy('scf',return_wfn=True)
+  scf_wfn.to_file('my_wfn')
+  set guess read
+  energy('scf', restart_file='my_wfn')
+
+Setting the orbital guess explicitly to ``read`` is crucial as this is not done automatically when 
```

Comment:
Why not? Is there ever a case where a user is going to supply this argument and _not_ want to read the orbitals?

Reply:
Not sure. I stayed with the old behaviour initially but I think I would also favour automatically turning on `READ`

Reply:
Next commit changes the behaviour to automatically enable `guess read` for the local scope.